Lib/fontTools/ttLib/tables/_g_l_y_f.py

```diff
@@ -1299,6 +1377,8 @@ def trim(self, remove_hinting=False):
                 i += 2 + instructionLen
             # Remove padding
             data = data[:i]
+        elif self.isVarComposite():
+            raise NotImplementedError
```

Comment:
if varComposites cannot (unlike simple and regular composite glyphs) have hinting instructions, should this `pass` or return early instead of raising NotImplementedError? Just a few lines above within the same `trim` method you do:

```pyhon
                elif self.isVarComposite():
                    pass  # Doesn't have hinting
```

Reply:
But this is about removing padding, not just hinting. It can and should eventually be implemented. I'll get on implementing it.